k8s/upgrades/0.8.0-0.8.1/cstor-pool-patch.tpl.json

```diff
@@ -0,0 +1,50 @@
+{
+  "metadata": {
+     "labels": {
+        "openebs.io/cas-template-name": "cstor-pool-create-default-0.8.1",
```

Comment:
CAS template should **not** be updated.
It represents the template that was used during creation of this resource.

Reply:
Doubt: Will all the unchanged labels remain as-is or will they get replaced.

Reply:
it will remain as-is

Reply:
It is good practice to remove from the patch.